### ISSUE TITLE
Subtitle viewer: a couple of things that could be improved

### ISSUE THREAD

**Issue by notargets** [xylosper/bomi#328]
There are a few issues with the subtitle viewer I find in the latest stable version of bomi:

1. When I open the subtitle viewer, the start and end columns are so narrow that the time codes don't fit in. When I change their width, their size is remembered only as long as bomi is running.
2. The "Scroll to current time" option doesn't seem to work even if it's enabled.
3. When I type something in the "Caption" field and I double click on a line that appears in the filtered list, it only jumps to the very first line in the subtitle.
4. I'm using a dark colour theme and the text in the "Start" and "End" fields are black, which makes it very hard to read it. (Everything else including what I enter into the "Caption" field follow the colour setting I set system-wide, namely white.)

Since these are relatively minor things, I thought it might be better to mention all of them in a single ticket rather than creating individual ones for each of them. I hope you don't mind.

**Comment by xylosper** [comment]
Fixed first 3 bugs.

For wrong styled color, can you post a screenshot?
I couldn't unserdantd waht's going on here because I don't think I changed any style in the dialog.

**Comment by notargets** [comment]
What I mean is that I changed the colour scheme in the settings of KDE so that all (or most) applications should have a dark background and light text. bomi respects this colour scheme too, except for what I type in the field "Start" and "End", so the text remains black there.

In the screenshot below, I typed "00:40:00" in the "Start" field but it is hard to see due to the low contrast between the colours. The rest of the text appearing in the window is fine.

![bomi_subtitle_viewer](https://cloud.githubusercontent.com/assets/10419911/7332564/2e11c836-eb45-11e4-983b-06638222233d.png)
